Server/User/utils/encryption_utils.py

```
@@ -43,13 +43,29 @@ def _get_or_create_key(self) -> bytes:
+            # C# Fernet library only accepts standard base64 keys, not URL-safe
+            # So we convert URL-safe to standard base64 for compatibility
             try:
+                # Try URL-safe base64 first (from environment variable)
                 decoded = base64.urlsafe_b64decode(value)
                 if len(decoded) == 32:
+                    # Convert to standard base64 for compatibility with C# Fernet library
+                    # Python Fernet accepts both formats, but we use standard for consistency
+                    standard_key = base64.b64encode(decoded).decode("utf-8")
+                    return standard_key.encode("utf-8")
+            except Exception:
+                pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)